csw-location/src/main/scala/csw/services/location/models/TmtSerializable.scala

```diff
@@ -1,6 +1,6 @@
 package csw.services.location.models
 
 /**
-  * A Serializable for Kryo hierarchy to work
+  * All models that resides in CRDT has to be serializable using TmtSerializable. It is required since they have to be communicable over the network.
```

Comment:
Not related to CRDT or cluster, but basically anything to with remote akka actors and message passing.